api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -441,11 +441,42 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 	@Test
 	public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		Visit visit = makeVisit(42);
-		visit
-		        .setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
-		
+		visit.setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
 		Errors errors = new BindException(visit, "visit");
 		new VisitValidator().validate(visit, errors);
 		assertEquals(true, errors.hasFieldErrors("voidReason"));
 	}
+
+	@Test
+	public void validate_shouldFailIfStartDateTimeIsBeforeBirthDateWhereBirthDateIsNotEstimated() {
+		Calendar calendar = Calendar.getInstance();
+		Visit visit = new Visit();
+		visit.setPatient(Context.getPatientService().getPatient(2));
+		visit.getPatient().setBirthdate(calendar.getTime());
+		calendar.set(2000, 9,8);
+		visit.setStartDatetime(calendar.getTime());
+		Errors errors = new BindException(visit, "visit");
+		new VisitValidator().validateBirthDate(visit, errors);
```

Comment:
after you changed your new method to private this just changes to `new VisitValidator().validate(visit, errors);` like in the other tests

I would also suggest you add newlines in between blocks of code that belong together to make it easy to read.

I like the AAA (arrange, act, assert) way which would make your code like

```java
public void validate_shouldFailIfStartDateTimeIsBeforeBirthDateWhereBirthDateIsNotEstimated() {

Calendar calendar = Calendar.getInstance();
...
Errors errors = new BindException(visit, "visit");

new VisitValidator().validateBirthDate(visit, errors);

assertEquals(true, errors.hasFieldErrors("startDatetime"));
```

the reader can easily see whats there to setup the test, what is actually tested and what are your expectations on the outcome at the end.